src/test/java/lesson7/task2/TextConverterTest.java

```diff
@@ -0,0 +1,24 @@
+package lesson7.task2;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TextConverterTest {
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = {"first text", "second text", "third text"})
+    void shouldCheckIfTextIsConvertedToCapital(String text) {
```

Comment:
positive and negative scenario check in one test. should be splitted into separate methods or refactored